webapp/WEB-INF/templates/admin/plugins/blog/insertservice/select_blog_link.html

```diff
@@ -0,0 +1,149 @@
+<script src="js/admin/jquery/jquery-3.6.0.min.js"></script>
```

Comment:
isn't it already included in lutece headers ?

Reply:
Hi, the pop-up window displaying the _Lutece InsertServices_ may not include this lib.

As a result, the input fields requiring the use of a calendar are not properly handled (no calendar view and no date selection are displayed).